README.md

```diff
@@ -58,14 +67,14 @@ Group](https://github.com/nodejs/Release).
 | 16.x    | :x:                |
 
 ## Methods
-### With npm
+### Using npm
 
 The CLI is built with Node.js and can be installed using `npm`. In order
 to use this method, you must have `node` and `npm` installed on your
-system already.
+system already. You can find download instructions [here](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm). To download run the following command:
```

Comment:
If we can limit lines to 80 characters where possible it makes it easier to edit and read